### PR TITLE
Update beats framework to a314e34

### PR DESCRIPTION
**Pull request by roncohen**

### CODE REVIEW

**Comment by roncohen** [comment]
ran into this now: `failed building apm-server-oss type=dmg for platform=darwin/amd64: failed to build internal beat pkg: copy failed: cannot stat source file apm-server.reference.yml: stat apm-server.reference.yml: no such file or directory` cc @andrewkroh

File: _beats/vendor/vendor.json [hunk 1]
```diff
@@ -521,7 +511,7 @@
 		},
 		{
 			"path": "github.com/elastic/go-structform/internal/visitors",
-			"revision": "",
+			"revision": "v0.0.4",
```
Comment (simitt) [inline _beats/vendor/vendor.json]: This looks strange, also I could not find `github.com/elastic/go-structform/internal/visitors`, but `github.com/elastic/go-structform/visitors`. Same for `github.com/elastic/go-structform/internal/ubjson` above. 

That looks like a potential issue in beats though, https://github.com/elastic/beats/pull/7676

**Comment by simitt** [comment]
oh, just saw your comment above now.

**Comment by roncohen** [comment]
Added a commit to fix the problem mentioned above `apm-server.reference.yml: no such file or directory`.

**Pull request merged**